graphql_compiler/cost_estimation/statistics.py

```diff
@@ -101,22 +101,29 @@ def get_histogram(self, vertex_name, field_name):
 class LocalStatistics(Statistics):
     """Provides statistics using ones given at initialization."""
     def __init__(
-        self, class_counts, edge_count_between_vertex_pairs, count_of_distinct_field_values,
-        histograms
+        self, class_counts, edge_count_between_vertex_pairs=None,
+        count_of_distinct_field_values=None, histograms=None
     ):
         """Initializes statistics with the given data.
 
         Args:
             class_counts: dict, str -> int, mapping vertex/edge class name to class count.
-            edge_count_between_vertex_pairs: dict, (str, str, str) -> int,
-                mapping tuple of (vertex out class name, vertex in class name, edge class name) to
+            edge_count_between_vertex_pairs: optional dict, (str, str, str) -> int, mapping 
+                tuple of (vertex source class name, edge class name, vertex target class name) to
                 count of edge instances of given class connecting instances of two vertex classes.
-            count_of_distinct_values: dict, (str, str) -> int, mapping vertex class name and field
-                name on that vertex class to the count of distinct values of the field for that
-                vertex class.
-            histograms: dict, (str, str) -> list[tuple(float, float, int)], mapping vertex class
-                name and field name on that vertex class to histogram.
+            count_of_distinct_values: optional dict, (str, str) -> int, mapping vertex class name
+                and field name on that vertex class to the count of distinct values of the field for
+                that vertex class.
+            histograms: optional dict, (str, str) -> list[tuple(float, float, int)], mapping vertex
```

Comment:
I'm confused about the histogram format. Why is each data point a tuple of floats and ints? Doesn't that implicitly assume that we are histogramming over a float-typed field?

Please explain the data format of the histogram. Alternatively, since we aren't using histograms yet, and we haven't implemented anything there, just delete all mentions of histograms in the API. When we actually need them and have the time to implement them, then we can have the design discussion around them.

Reply:
I did the latter.